chart/assets/operations/instance_groups/api.yaml

```diff
@@ -32,6 +32,27 @@
   path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/ccdb/connection_validation_timeout?
   value: 3600
 
+# Default settings for the CF info endpoints.
+##
+# Note that the version information has to be a plain integer, as per the
+# Membrane::SchemaValidation occuring in `bosh-pre-start-cloud-controller-ng`.
+
+- type: replace
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/name?
+  value: 'SUSE CAP Kubecf'
+- type: replace
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/build?
+  value: {{ default .Chart.Version .Chart.AppVersion | quote }}
+- type: replace
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/version?
+  value: 2
+- type: replace
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/support_address?
+  value: 'https://scc.suse.com'
```

Comment:
This would also be incorrect for KubeCF (SUSE shouldn't be getting support requests for things we don't sell).

Reply:
Tweaked in commit 6997dff1ed9dd3f7eef0472ad4ea5ffe04afb737 as FIXME.

Reply:
Fixed with 313c1b19f22c980f80ff3cd094f1ffd811409410